plugins/cassandra/cpp-driver

```diff
@@ -1 +1 @@
-Subproject commit d7bbad34db39a51f209c6fadd07c9ec3a0bb86b7
```

Comment:
I thought this might be there, but couldn't see which commit it was in - I'll search